src/main/scala/org/ergoplatform/ErgoApp.scala

```diff
@@ -166,28 +165,7 @@ class ErgoApp(args: Args) extends ScorexLogging {
     val bindAddress = settings.restApi.bindAddress
 
     Http().newServerAt(bindAddress.getAddress.getHostAddress, bindAddress.getPort).bindFlow(httpService.compositeRoute)
-
-    //on unexpected shutdown
-    Runtime.getRuntime.addShutdownHook(new Thread() {
-      override def run() {
-        log.error("Unexpected shutdown")
-        stopAll()
-      }
-    })
```

Comment:
Why this hook is not necessary?

Reply:
I ported it to a CoordinatedShutdown task which is triggered on this hook implicitly ... One would have to explicitly disable it : 
```
akka.coordinated-shutdown.run-by-jvm-shutdown-hook=off
```